build

```diff
@@ -7,6 +7,12 @@ GO="godep go"
 ORG_PATH="github.com/kelseyhightower"
 REPO_PATH="${ORG_PATH}/confd"
 
+export GOPATH=${PWD}/gopath
+
+rm -f $GOPATH/src/${REPO_PATH}
```

Comment:
This is concerning if you're running this script via $HOME and you have your normal $GOPATH set to `gopath`:

```
$ pwd
/Users/bacongobbler
$ ls | grep gopath
gopath
$ ./gopath/src/github.com/kelseyhightower/confd/build
```

Which will delete your confd changes.